flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/rules/logical/subquery/SubqueryCorrelateVariablesValidationTest.scala

```diff
@@ -155,8 +154,6 @@ class SubqueryCorrelateVariablesValidationTest extends SubQueryTestBase {
         |FROM   t1
         |    WHERE  t1a = 'test'
       """.stripMargin
-    assertThatExceptionOfType(classOf[AssertionError])
-      .isThrownBy(() => util.verifyRelPlan(sqlQuery))
```

Comment:
Seems since some time it was fixed and this code was continuing throwing exception with a different reason (nothing in xml plans... ). So generated plans for this